src/components/Vehicle.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { VehicleContainer, VehicleLabel, Button } from "./styled";
+import VehicleStatusSelector from "./VehicleStatusSelector";
+
+const Vehicle = (props) => {
+  const {
+    stationId,
+    vehicle,
+    handleVehicleStatusChange,
+    showMoveButton,
+    handleMoveVehicle
+  } = props;
+
+  if (!vehicle) return null;
+
+  return (
+    <VehicleContainer>
+      <VehicleLabel>Vehicle #{vehicle.id}</VehicleLabel>
+      <VehicleStatusSelector
+        stationId={stationId}
+        vehicle={vehicle}
+        handleVehicleStatusChange={handleVehicleStatusChange}
+      />
+
+      {stationId !== "4" && (
```

Comment:
💡 There are no other stations after station `4` i.e. our delivery truck. Hence we hide the button for moving the vehicle.